shared.csv

```diff
@@ -1,33 +1,57 @@
 name,quadrant,ring,isNew,description
-Kotlin,Languages & Frameworks,Trial,TRUE,
-TypeScript,Languages & Frameworks,Trial,TRUE,
-Spring Security,Languages & Frameworks,Trial,TRUE,
-Kibana,Tools,Trial,TRUE,
-Elasticsearch,Infrastructure Components,Trial,TRUE,
-Filebeat,Tools,Trial,TRUE,
-Yarn,Tools,Trial,TRUE,
-Docker,Infrastructure Components,Trial,TRUE,
-GitHub Actions,Services,Trial,TRUE,
-Make,Tools,Trial,TRUE,
-Express,Languages & Frameworks,Trial,TRUE,
-Node.js,Languages & Frameworks,Trial,TRUE,
-React,Languages & Frameworks,Trial,TRUE,
-Redux,Languages & Frameworks,Trial,TRUE,
-Gradle,Tools,Trial,TRUE,
-Nimbus,Languages & Frameworks,Trial,TRUE,
-Postgresql,Infrastructure Components,Trial,TRUE,
-GraphQL,Languages & Frameworks,Assess,TRUE,At the moment there is no use case for our application but we'd like to use it at some point
-React i18next,Languages & Frameworks,Trial,TRUE,
-kustomize,Tools,Trial,TRUE,
-Kubernetes,Infrastructure Components,Trial,TRUE,
-Google Cloud Build,Services,Trial,TRUE,
-NextJS,Languages & Frameworks,Hold,TRUE,Benefits don't outweigh the effort for the time being
-JUnit 5,Languages & Frameworks,Trial,TRUE,
-GreenMail,Tools,Trial,TRUE,
+OKRs,Techniques,Adopt,FALSE,"OKRs stands for Objectives and Key Results. At the beginning of every quarter we plan which Objectives we want to accomplish. After we have agreed on which Objectives to tackle, we assign different Key Results to them which represent concrete steps or actions on how to achieve set Objectives."
+DRIs,Techniques,Adopt,FALSE,"DRI stands for Directly Responsible Individual. A DRI is responsible for a quarterly objective being completed, but they do not need to do (all) the work themselves. Each team member can be the DRI for one or multiple objectives each quarter."
+1:1,Techniques,Adopt,FALSE,"Regular 1:1 meetings with the supervisor"
```

Comment:
@dlavrenuek What do you think of adding more detail - also about what the purpose of the meetings is?

```suggestion
1:1,Techniques,Adopt,FALSE,"Our developers have weekly 1:1 meetings with the Engineering Manager that provide each of us up to an hour of time to improve continuously and to give each other feedback. We also do skip-level 1:1's that skip one level of the hierarchy at a slower schedule."
```